### ISSUE TITLE
Data tab does not open on click

### ISSUE THREAD

**Issue by st-h** [emberjs/ember-inspector#2185]
**Describe the bug**
When I open the ember inspector and try to click the data tab, nothing happens. The previously selected tab is still displayed. I do not see any error within the console.

**Environment**
Ember Inspector | 4.7.0
Ember | 4.5.1
Ember Data | 4.6.1

Tested Browsers (same issue with both): 
Chrome 105.0.5195.125 (Official Build) (x86_64)
Firefox Developer Edition 106.0b9 (64-bit)

**Comment by patricklx** [comment]
This might also cause it: https://github.com/emberjs/data/issues/8205

**Comment by st-h** [comment]
Thank you. I always observed this issue when there were items in the store. I just tried out version `3.8.0`. Now the store tab opens fine, however it does not show any data. I found out that there is one way to workaround this: Open the data tab. Then the requests to load the data need to be made while the data tab is open and the data does appear just fine. Data is not shown if the data tab is opened after data has been loaded.

**Comment by rwwagner90** [comment]
> Thank you. I always observed this issue when there were items in the store. I just tried out version `3.8.0`. Now the store tab opens fine, however it does not show any data. I found out that there is one way to workaround this: Open the data tab. Then the requests to load the data need to be made while the data tab is open and the data does appear just fine. Data is not shown if the data tab is opened after data has been loaded.

@patricklx that sounds like a major bug, should we revert or do you have a fix in mind? We should write some tests for this.

**Comment by patricklx** [comment]
@rwwagner90 
> This might also cause it: [emberjs/data#8205](https://github.com/emberjs/data/issues/8205)

That is the issue causing it

**Comment by patricklx** [comment]
@st-h which ember data version are you using. Affected version should be >= 4.5.0

**Comment by st-h** [comment]
@patricklx I am on ember data 4.6.1

**Comment by rwwagner90** [comment]
So nothing to fix here?

**Comment by patricklx** [comment]
No, nothing we can do here. Would be best if ember data also backports the fix to 4.5 and 4.6
